listenbrainz/domain/critiquebrainz.py

```diff
@@ -25,76 +17,18 @@
 CRITIQUEBRAINZ_REVIEW_LICENSE = "CC BY-SA 3.0"
 
 
-class CritiqueBrainzService(ExternalService):
+class CritiqueBrainzService(BaseBrainzService):
 
     def __init__(self):
-        super(CritiqueBrainzService, self).__init__(ExternalServiceType.CRITIQUEBRAINZ)
-        self.client_id = current_app.config["CRITIQUEBRAINZ_CLIENT_ID"]
-        self.client_secret = current_app.config["CRITIQUEBRAINZ_CLIENT_SECRET"]
-        self.redirect_uri = current_app.config["CRITIQUEBRAINZ_REDIRECT_URI"]
-
-    def add_new_user(self, user_id: int, token: dict) -> bool:
```

Comment:
Why is all this code no longer needed?

Reply:
Moved to brainz_service.py as it is commonly used between CB and MB auth.